docs/public/_redirects

```diff
@@ -7,6 +7,7 @@
 /r/x-get-license https://mui.com/pricing/ 302
 /r/x-license-eula https://mui.com/store/legal/mui-x-eula/ 302
 /r/x-license-key-installation https://mui.com/x/advanced-components/#license-key-installation 302
+/r/x-grid-no-dimensions https://mui.com/x/react-data-grid/layout/
```

Comment:
What do you think of this format? It seems more future-proof if we link other errors.

```suggestion
/r/x-data-grid/no-dimensions https://mui.com/x/react-data-grid/layout/
```

Reply:
+1 to use x-data grid over x-grid, it makes things clearer. Regarding the subfolder, I think that it would introduce confusing, it's a redirection slug, hence I think that it should stay flat, e.g. https://github.com/vercel/next.js/tree/1628e6d16c35d16506a1be4fe2a3991b0e3046e1/errors.